NeuralNetworks/Practice/Questions.md

```diff
@@ -0,0 +1,37 @@
+<details>
+<summary>What are neural networks generally used for?</summary>
+<br>
+Neural networks are generally used to identify patterns or model complex relationships between inputs and outputs in datasets.
+</details>
+
+---
+
+<details>
+<summary>How can a GPU help with neural networks?</summary>
+<br>
+
+A GPU can be used to increase the speed at which the neural network trains.
+
+</details>
+
+---
+
+<details>
+<summary>What line of code is used to ensure that your user has access to the GPU?</summary>

```

Comment:
"line of code" seems kinda misleading, maybe "command" instead
Also I don't know if this question is appropriate for this section as it is about setting up ROCm and not neural networks

Reply:
I changed it to command. I think it is important as its the only part that actually relates to the GPU kit.